src/streams/StreamItem.js

```diff
@@ -48,13 +48,13 @@ type Props = $ReadOnly<{|
  * @prop description - the stream's description
  * @prop isMuted - false for a Stream; !sub.in_home_view for Subscription
  * @prop isPrivate - .invite_only for a Stream or a Subscription
+ * @prop isSubscribed - whether the user is subscribed to the stream
  * @prop color - if provided, MUST be .color on a Subscription
  * @prop backgroundColor - if provided, MUST be .color on a Subscription
  *
  * @prop unreadCount - number of unread messages
  * @prop iconSize
  * @prop showSwitch - whether to show a toggle switch (ZulipSwitch)
- * @prop isSwitchedOn - initial value of the toggle switch, if present
```

Comment:
🎉

I'd put this `isSwitchedOn` -> `isSubscribed` change into its own NFC commit, so the pure refactor can stand and be verified on its own.

Reply:
Good call, done.